apps/admin-ui/src/i18n/messages.ts

```diff
@@ -1894,11 +1894,31 @@ const translations: ITranslations = {
     reserveeLastName: ["Sukunimi"],
     addressStreet: ["Lähiosoite"],
     addressCity: ["Kotikunta"],
+    addressCountry: ["Maa"],
+    addressForeign: ["Osoite ulkomailla"],
+    addressAdditional: ["Lisätiedot"],
     reserveeEmail: ["Sähköpostiosoite"],
     reserveePhone: ["Puhelin"],
     reserveeType: ["Asiakastyyppi"],
     reserveeOrganisationName: ["Yhdistyksen nimi"],
     reserveeBusinessName: ["Yrityksen nimi"],
+    reserveeShowInfosNotification: [
+      "Tiedot haetaan Helsinki-profiilista ja sen kautta Väestötietovirastolta.",
+    ],
+    reserveeUnableToShowData: ["Tietoja ei voida näyttää"],
+    reserveePermissionError: [
+      "Käyttöoikeutesi eivät riitä henkilötietojen näyttämiseen.",
+    ],
+    reserveeKeycloakExpiredError: [
+      "Keycloakin päivitystunnus on vanhentunut. Kirjaudu ulos ja takaisin sisään.",
```

Comment:
The second sentence is the important part while the first will in any case be gibberish of some sort to most users, but perhaps it could be rewritten as (just a hair less obscurely, although the underlying reason is omitted; "uudistettua" would then be more accurate but perhaps too technical a distinction): "Henkilötietoja ei voida näyttää, koska niiden katselun vaatimaa käyttöoikeustunnusta ei saatu noudettua. Kirjaudu ulos ja takaisin sisään sen virkistämiseksi."